python_modules/dagster/dagster/_core/run_coordinator/queued_run_coordinator.py

```diff
@@ -204,6 +237,35 @@ def config_type(cls) -> UserConfigSchema:
                     " launcher is being used."
                 ),
             ),
+            "block_op_concurrency_limited_runs": Field(
+                config=bool,
+                is_required=False,
+                description=(
+                    "Determines whether or not a run will be dequeued if it consists of ops that "
+                    "will all be blocked waiting for global op concurrency slot to be free."
+                ),
+            ),
+            "op_concurrency_slot_offset": Field(
```

Comment:
naming is hard but i find this name pretty confusing - this is basically the number of additional slots we pretend each step has when deciding whether or not it will be full right? I know we use "buffer" below, but maybe op_concurrency_slot_buffer? It seems like more of of a buffer than an offset to me